src/jitlayers.cpp

```diff
@@ -1665,6 +1665,17 @@ void fixupTM(TargetMachine &TM){
     }
 }
 
+extern int jl_opaque_ptrs_set;
+void SetOpaquePointer(LLVMContext &ctx) {
+    if (jl_opaque_ptrs_set)
+        return;
+#ifndef JL_LLVM_OPAQUE_POINTERS
```

Comment:
Is anyone actually using this build flag, given we have a runtime option for setting it as well? I'm in favor of just deleting.